### PR TITLE
Porting of social.md

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/software/social.md [hunk 1]
```diff
@@ -28,3 +24,227 @@
...
+Also Installs:
+
+- Webserver
+
+![DietPi social software phpBB](../assets/images/dietpi-software-social-phpbb.png){: style="width:500px"}
+
+### Access Forums
+
+URL = `http://<your.IP>/phpBB3`  
+(from DietPi v6.33 on: `http://<your.IP>/phpbb`)
```
Comment (MichaIng) [inline docs/software/social.md:58]: I added the DietPi v6.33 info during dev stage. Now that it's released, let's only show the new `/phpbb` location here, same for a few cases below.

Comment (MichaIng) [inline docs/software/social.md:58]: I added the DietPi v6.33 info during dev stage. Now that it's released, let's only show the new `/phpbb` location here, same for a few cases below.

File: docs/software/social.md [hunk 2]
```diff
@@ -28,3 +24,227 @@
...
+=== "Access to the web interface"
+
+    URL = `http://<your.IP>/wordpress`
+
+=== "First time connect"
+
+    The first time you connect, enter the following MySQL database details:
+
+    - Database name = `wordpress`
+    - Username = `wordpress` (Previous to DietPi v6.0 = `root`)
```
Comment (MichaIng) [inline docs/software/social.md:132]: Let's not add ancient info anymore here and remove `(Previous to DietPi v6.0 = `root`)`.

File: docs/software/social.md [hunk 3]
```diff
@@ -28,3 +24,227 @@
...
+
+      ```sh
+      mkdir /var/www/gallery/MyImageFolder
+      ```
+
+    - Put an image file inside your new folder
+
+      ```sh
+      wget http://media.moddb.com/cache/images/articles/1/153/152601/crop_120x90/download.png
+      mv download.png /var/www/gallery/MyImageFolder/
```
Comment (MichaIng) [inline docs/software/social.md:165]: Easier:
```
wget http://media.moddb.com/cache/images/articles/1/153/152601/crop_120x90/download.png -O /var/www/gallery/MyImageFolder/download.png
```
At least let's not use a non-HTTPS resource as example (although Tr-Zero is from Dan's Fuzon Design), but our logo instead: https://dietpi.com/images/dietpi-logo_192x192.png
But also much longer.

**Review by MichaIng — CHANGES_REQUESTED** [review]

File: docs/software/social.md (hunk 1 above)
Comment (StephanStS) [reply docs/software/social.md:58]: Ok. Deleted all text passages referring to old DietPi versions.

File: docs/software/social.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/social.md:132]: Ok. Deleted.

File: docs/software/social.md (hunk 3 above)
Comment (StephanStS) [reply docs/software/social.md:165]: I changed it to:

      ```sh
      wget https://dietpi.com/images/dietpi-logo_192x192.png  
      mv dietpi-logo_192x192.png /var/www/gallery/MyImageFolder/
      ```

I would rather keep the two commands, for most of the readers I assume it's easier for them to understand what happens.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**